packages/paste-website/src/pages/patterns/confirmations/index.mdx

```diff
@@ -0,0 +1,162 @@
+export const meta = {
```

Comment:
todo: remove this page since it's already live on the doc site.

Reply:
thank you!! might want to sync with Nora on this. She'd mentioned she was going to try to do this when we met last week.

Reply:
fixed!